### PR TITLE
Testing and other updates

### PR DESCRIPTION
**Pull request by axelrtgs**
Do Not merge, local testing is failing due to system issues using this PR to take advantage of Travis testing.

will update when work complete

### CODE REVIEW

**Comment by damacus** [comment]
We probably want to add appveyor testing for this one.

File: recipes/default.rb [hunk 1]
```diff
@@ -1,118 +1,118 @@
...
+  backup false
+  rights :read, node['winrm']['Everyone_Group']
+end
+
+ruby_block 'read-winrm-thumbprint' do
+  block do
+    f = File.open("#{Chef::Config[:file_cache_path]}\\winrm.thumbprint", 'r')
+    val = f.read.strip
+    f.close
+    node.override['winrm']['thumbprint'] = ((val.empty? || val.nil?) ? nil : val)
```
Comment (damacus) [inline recipes/default.rb]: node.default if possible please 😿 
/cc @tas50

Comment (axelrtgs) [reply recipes/default.rb]: fixed pushing now

**Comment by axelrtgs** [comment]
Closing as no longer relevant due to not using travis.

**Pull request closed without merging**